image-swipe.d.ts

```diff
@@ -17,6 +17,11 @@ import { EventData } from "data/observable";
 import { CoercibleProperty, Property } from "ui/core/view";
 import { ItemsSource } from "ui/list-picker";
 import { ScrollView } from "ui/scroll-view";
+import { Cache } from "ui/image-cache";
+
+export class ImageSwipeBase {
```

Comment:
Ok, +1 for the later approach. I think its more a typescript issue then {N} one.